src/core/annotation.js

```diff
@@ -1282,6 +1285,15 @@ class WidgetAnnotation extends Annotation {
     const totalHeight = this.data.rect[3] - this.data.rect[1];
     const totalWidth = this.data.rect[2] - this.data.rect[0];
 
+    if (!this.data.defaultAppearance) {
+      // The DA is required and must be a string.
+      // There is no reason to have a font named Helvetica in
+      // the resource dictionary so the evaluator will fallback on
+      // a default one: it isn't ideal but at least it prevents from
+      // exceptions and it allows to save or print the file as expected.
+      this.data.defaultAppearance = "/Helvetica 10 Tf 0 g";
```

Comment:
Looking at https://stackoverflow.com/questions/17503349/da-pdfbox-how-to-create, would using 0 perhaps make sense here so it's auto-scaled?